modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -435,7 +436,7 @@ export class Wallets implements IWallets {
           throw new Error('cannot generate backup keypair without passphrase');
         }
         // No provided backup xpub or address, so default to creating one here
-        return this.baseCoin.keychains().createBackup({ reqId });
+        return this.baseCoin.keychains().createBackup({ reqId, passphrase: params.passphrase });
```

Comment:
Is this the only call-site we need to update?

Reply:
yes